dev_glue.py

```diff
@@ -7,9 +7,6 @@
 See __path__ documentation at:
     https://docs.python.org/2/tutorial/modules.html
 """
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
 
 import candig
-candig.__path__.insert(0, 'candig')
+candig.__path__.append('candig')
```

Comment:
Was there any benefit to have it in the front?

Reply:
No, though the `__path__` no longer has `insert` attribute because it is not a `list`.

Reply:
Actually, candig might not be a module anymore. This needs further investigation.